docs/cimpl6UserGuide.md

```diff
@@ -133,7 +140,15 @@ CIMPL Compilation Errors are structured in the following format:
 
 A detailed list of CIMPL compilation errors and troubleshooting suggestions are available **[here](https://github.com/standardhealth/shr-cli/wiki/Error-Message-Documentation)**.
 
-## IG Publisher Output
+## FHIR IG Publisher Output
+
+The final step in the IG creation process is to run the **[FHIR IG Publisher](http://wiki.hl7.org/index.php?title=IG_Publisher_Documentation)**. This tool is maintained and owned by HL7 FHIR. 
+
+At a command prompt, enter (changing mcodeR4 to your CIMPL model name, and the -ig name to an appropriate value):  (Mark & May)
+
+* yarn run ig: publish   QUESTION: readme has this - this needs work 
```

Comment:
yarn run ig: publish is used if you have taken the defaults for running `shr-cli` (which I think puts the output into the same directory as the source code.  The more extensive IG Publisher command allows you to generate the IG output from any directory.